projects/app/src/client/ui/RectButton.tsx

```
@@ -1,89 +1,204 @@
-            ]}
-          >
+      <View
+        className={roundedRect({
+          flat,
+          variant,
+          disabled,
+          hoveredOrPressed: pressed || hovered,
+          className,
+        })}
+      >
+        {typeof children === `string` ? (
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)